server/vulnerabilities/macoffice/analyzer.go

```diff
@@ -178,12 +180,12 @@ func Analyze(
 
 		detected := collectVulnerabilities(software, product, relNotes)
 		// The 'software' instance we get back from the iterator does not include vulnerabilities...
-		existing, err := getStoredVulnerabilities(ctx, ds, software.ID)
+		existing, err := getStoredVulnerabilities(ctx, ds.SoftwareByID, software.ID)
 		if err != nil {
 			return nil, err
 		}
 
-		inserted, err := updateVulnsInDB(ctx, ds, detected, existing)
+		inserted, err := updateVulnsInDB(ctx, detected, existing, ds.DeleteSoftwareVulnerabilities, ds.InsertSoftwareVulnerabilities)
```

Comment:
Same here.

Reply:
Addressed in https://github.com/fleetdm/fleet/pull/10301/commits/1d763092409617af5e52bccf284c46765fe3ff14